### PR TITLE
Assign WIKIDATAIDs for Extreme distance candidates (Issue #460)

### PR DESCRIPTION
**Pull request by mizmay**
Update the WIKIDATAIDs based on @elliotap's recommendations in #460.

In these cases I implemented the wikidataid candidate in the issue, but held off on the suggested alternative WIKIDATAID, because I think counties, districts belong in a different dataset:
- Kotabumi
- Pati
- Shangdu
- Shashi

If I'm not applying the right heuristic, please confirm the suggested alternatives for these cities and I will change them.

(NOTE: NVK fixed link to issue.)

### CODE REVIEW

File: tools/515_extreme_distance_candidates/Update_extreme_distance_candidates.R [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  # Chanaral -> Chañaral
+  rows_update(by = "NE_ID", tibble(NE_ID = 1159148559,
+                                   NAME = "Chañaral",
+                                   NAMEALT = "Chanaral",
+                                   NAMEASCII = "Chanaral",
+                                   # move geometry to -26.34810,-70.62147
+                                   LATITUDE = -26.34810,
+                                   LONGITUDE = -70.62147,
+                                   WIKIDATAID = "Q3763")) %>%
+  # Fuyu -> Songyuan
```
Comment (nvkelso) [inline tools/515_extreme_distance_candidates/Update_extreme_distance_candidates.R:66]: Should also have population change to `495900` with pop_rank changes?

File: tools/515_extreme_distance_candidates/Update_extreme_distance_candidates.R [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  rows_update(by = "NE_ID", tibble(NE_ID = 1159141037,
+                                   NAME = "Songyuan",
+                                   NAMEASCII = "Songyuan",
+                                   NAMEALT = "Fuyu",
+                                   WIKIDATAID = "Q185940")) %>%
```
Comment (nvkelso) [inline tools/515_extreme_distance_candidates/Update_extreme_distance_candidates.R:71]: To `Q92364`

**Comment by nvkelso** [comment]
> In these cases I implemented the wikidataid candidate in the issue, but held off on the suggested alternative WIKIDATAID, because I think counties, districts belong in a different dataset:
> Kotabumi
> Pati
> Shangdu
> Shashi
> If I'm not applying the right heuristic, please confirm the suggested alternatives for these cities and I will change them.

Natural Earth is a little weird / expansive of it's view of populated places so @elliotap's suggestions stand for the above. I'll note them as specific PR comments. 

But for practical purposes they will probably be 1-off followups as this PR is merged mostly as-is and https://github.com/nvkelso/natural-earth-vector/issues/5 is reapplied.

**Comment by mizmay** [comment]
Thanks for the double confirmation! Latest commit implements all of @elliotap's recommendations, I was uncertain what the pop rank for Fuyu should change to so only added the population value.

Indeed each of these scripts enforces some things about field lengths and types ;). Happy to revisit some day! In the meantime, please merge if you are satisfied or ready to move on to just manual/one-off fixes, if not I'll come back for another round when I have wifi.

:)

**Review by nvkelso — APPROVED** [review]
LGTM

**Pull request merged**